core/embed/trezorhal/stm32f4/touch/ft6x36.c

```diff
@@ -376,8 +390,14 @@ uint32_t touch_get_event(void) {
   uint8_t flags = regs[FT6X63_REG_P1_XH] & FT6X63_EVENT_MASK;
 
   // Extract touch coordinates
-  uint16_t x = ((regs[FT6X63_REG_P1_XH] & 0x0F) << 8) | regs[FT6X63_REG_P1_XL];
-  uint16_t y = ((regs[FT6X63_REG_P1_YH] & 0x0F) << 8) | regs[FT6X63_REG_P1_YL];
+  uint16_t x_raw =
+      ((regs[FT6X63_REG_P1_XH] & 0x0F) << 8) | regs[FT6X63_REG_P1_XL];
+  uint16_t y_raw =
+      ((regs[FT6X63_REG_P1_YH] & 0x0F) << 8) | regs[FT6X63_REG_P1_YL];
+
+  uint16_t x, y;
+
+  ft6x36_panel_correction(x_raw, y_raw, &x, &y);
```

Comment:
I would rather see a function here with just two arguments, so it can modify or keep the value. This solution with four arguments is a bit error-prone, but it's not a big deal.